CPU_consumption/Consumption/TotalConsumption.cpp

```diff
@@ -22,12 +22,12 @@ TotalConsumption::TotalConsumption(const struct Rect& currentArea)
 float TotalConsumption::calculate(unsigned long long idleTicks,
                                   unsigned long long totalTicks)
 {
-    unsigned long long totalTicksSinceLastTime = totalTicks - _previousTotalTicks;
-    unsigned long long idleTicksSinceLastTime  = idleTicks  - _previousIdleTicks;
-    float ans = 1.0f -
-                (totalTicksSinceLastTime > 0 ?
-                static_cast<float>(idleTicksSinceLastTime) / totalTicksSinceLastTime :
-                0);
+    auto totalTicksSinceLastTime = totalTicks - _previousTotalTicks;
+    auto idleTicksSinceLastTime  = idleTicks  - _previousIdleTicks;
+    float diff = static_cast<float>(idleTicksSinceLastTime) / totalTicksSinceLastTime;
```

Comment:
First and foremost, I had to cast it in order to avoid the integer division. 
I decided to use `static_cast` because it casts the primitive types (from `unsigned long long` to `float`). Of course, C-style cast could be used, but I just wanted to make my code look more modern.